.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: java
+jdk: oraclejdk8
+# Trusty dist needed for postgresql version 9.6
+dist: trusty
+# Caches downloaded libraries between builds
+cache:
+  directories:
+    - node_modules
+    - $HOME/.m2
+services:
+  - postgresql
+addons:
+  postgresql: '9.6'
+# Set the max_map_count for elasticsearch, as instructed by maven Elasticsearch plugin docs
+sudo: true
+before_script:
+  - sudo sysctl -w vm.max_map_count=262144
```

Comment:
I think you are right about the yum package: https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html